src/icons/heart-empty.svg

```diff
@@ -21,5 +21,35 @@
 			c8.547,24.385,12.164,50.811,10.75,78.542c-2.772,54.379-24.017,114.42-64.944,183.553
 			C773.338,635.262,656.457,747.659,489.322,855.248z"/>
 	</g>
-
+</g>

```

Comment:
This looks intersting, what is it?

Reply:
Its an heart svg which is not filled. Code looks a bit funky